cmd/webhook.go

```diff
@@ -220,9 +115,9 @@ func (o *webhookV2Options) validate(c *config.Config) error {
 	return nil
 }
 
-func (o *webhookV2Options) scaffolder(c *config.Config) (scaffold.Scaffolder, error) { //nolint:unparam
+func (o *webhookV2Options) scaffolder(c *config.Config) (scaffold.Scaffolder, error) { // nolint:unparam
 	// Load the boilerplate
-	bp, err := ioutil.ReadFile(filepath.Join("hack", "boilerplate.go.txt")) //nolint:gosec
+	bp, err := ioutil.ReadFile(filepath.Join("hack", "boilerplate.go.txt")) // nolint:gosec
```

Comment:
```suggestion
	bp, err := ioutil.ReadFile(filepath.Join("hack", "boilerplate.go.txt")) //nolint:gosec
```

Same as above

Reply:
One you suggested add the space and in the other remove. What is the rule?

Reply:
Anyway, I will keep it as the master since it not part of the scope of this PR.

Reply:
Both sugestions are removing, the master has them removed.